vm/compiler/src/program/finalize/command/mod.rs

```diff
@@ -67,8 +79,12 @@ impl<N: Network> FromBytes for Command<N> {
             1 => Ok(Self::Instruction(Instruction::read_le(&mut reader)?)),
             // Read the increment.
             2 => Ok(Self::Increment(Increment::read_le(&mut reader)?)),
+            // Read the load.
+            3 => Ok(Self::Load(Load::read_le(&mut reader)?)),
+            // Read the store.
+            4 => Ok(Self::Store(Store::read_le(&mut reader)?)),
             // Invalid variant.
-            3.. => Err(error(format!("Invalid command variant: {}", variant))),
+            5.. => Err(error(format!("Invalid command variant: {}", variant))),
```

Comment:
```suggestion
            _ => Err(error(format!("Invalid command variant: {}", variant))),
```

Reply:
Isn't the range syntax more suited for catching missing cases?